core/trino-spi/src/main/java/io/trino/spi/session/PropertyMetadata.java

```diff
@@ -249,6 +253,22 @@ public Object encode(T value)
                 object -> object);
     }
 
+    public static PropertyMetadata<List> listProperty(String propertyKey, String propertyDescription)
```

Comment:
This change is unnecessary. It's not clear it's an abstraction we want to add -- at least not in its current shape. Fro instance, it makes assumptions about the type of the values, case-sensitivity, etc.

Reply:
Reverted back